integration_tests/e2e/education-support-plan/create/checkYourAnswersChangeLinks.cy.ts

```diff
@@ -94,8 +96,30 @@ context(`Change links on the Check Your Answers page when creating an Education
       .thePlanWasCreatedBy('Joe Bloggs', 'Peer Mentor')
 
       // check and update other people consulted
+      .otherPeopleWereNotConsulted()
```

Comment:
This cypress test is about the Change links on Check Your Answers. It runs through the screen journey to add an Education Support Plan to get to Check Your Answers. And then for every question it clicks the Change link to change the answer.

The section in this test for "Other people consulted" was not complete because the original implementation did not include it on Check Your Answers.
This PR adds those 2 sections to Check Your Answers, and the changes here ensure that the Change links work as expected.